gio/README.md

```diff
@@ -6,7 +6,7 @@ GIO __2.56__ is the lowest supported version for the underlying library.
 
 ## Minimum supported Rust version
 
-Currently, the minimum supported Rust version is `1.80.0`.
+Currently, the minimum supported Rust version is `1.83.0`.
```

Comment:
Which part exactly needs the MSRV bump?

Reply:
The `ControlFlow::break_value` function, Rust 1.83 adds some functions to make `ControlFlow` more convenient.

Reply:
That's a simple match so we could also just not use it here. Updating the MSRV seems fine to me too though.

@bilelmoussaoui what do you think?

Reply:
Ok, then just waiting for @A6GibKm in the other thread, then I'll merge this.